### PR TITLE
Home page1

### PR DESCRIPTION
**Pull request by SanaTameem**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+<section>
+    <header>
+        <nav class="nav-bar">
+            <a href="#" class="nav-title">Welcome</a>
+            <a href="#" class="hamburger"><img src="Images/Enabled.svg" alt="hamburger"></a>
+        </nav>
```
Comment (divinecharlotte) [inline index.html:20]: - [ ] kindly consider having the hamburger menu alone in the nav tag for semantics reason

File: index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    </header>
+    <div class="hero">
+        <div class="hero-container">
```
Comment (divinecharlotte) [inline index.html:23]: - [ ] consider changing this div tag into a section tag to fulfill this requirement 👇 
> The container of the section is a `section` tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        <div class="social">
+            <a class="icon" href="#"><img src="Images/twitter (3).svg" alt="twitter-img"></a>
+            <a class="icon" href="#"><img src="Images/Linkedin icon (1).svg" alt="linkedIn-img"></a>
+            <a class="icon" href="#"><img src="Images/medium (1).svg" alt="medium"></a>
+            <a class="icon" href="#"><img src="Images/github.svg" alt="github-img"></a>
+            <a class="icon" href="#"><img src="Images/angellist.svg" alt="angellist-img"></a>
+        </div>
+        <div class="action-icon">
+            <a href="#"><img class="action" src="Images/Disabled.svg" alt=""></a>
+        </div>
```
Comment (divinecharlotte) [inline index.html]: - [ ] kindly consider the list of icons to use  a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @SanaTameem   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive Readme 👍 
- Good commit messages 👍 
- Good branch naming 👍 

## Required Changes ♻️

_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style1.css [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+.hero {
+  background-image: url(Images/header-illsutration-mobile@2x.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: auto;
+}
+
+.hero-container {
+  width: 100%;
```
Comment (amejid) [inline style1.css:48]: - [ ] [OPTIONAL] Currently, there is a small horizontal overflow on the page. Kindly fix it for a better user experience. I think removing `width: 100%;` will help in fixing the issue. 👍

**Review by amejid — APPROVED** [review]
Hi @SanaTameem, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing ✔️ 
- Good UI design ✔️ 
- Correct use of Github flow ✔️ 
## Optional suggestions
- I highly recommend adding a description to the PR to summarize the changes introduced in the pull request. Your PR does not have any description at the moment. Kindly update it for a more professional workflow.

Capture


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**